### ISSUE TITLE
配置了本地vllm部署的模型，但是实际评估的时候没有使用

### ISSUE THREAD

**Issue by 2niuhe** [justplus/llm-eval#12]
实际使用的是远端服务的模型，而不是我选择的本地模型

日志中大量类似错误：

2025-08-07 08:36:31,339 - app - WARNING - Skipping malformed model data from provider: {'id': 'neversleep/llama-3-lumimaid-70b', 'canonical_slug'
: 'neversleep/llama-3-lumimaid-70b', 'hugging_face_id': 'NeverSleep/Llama-3-Lumimaid-70B-v0.1', 'name': 'NeverSleep: Llama 3 Lumimaid 70B', 'crea
ted': 1715817600, 'description': "The NeverSleep team is back, with a Llama 3 70B finetune trained on their curated roleplay data. Striking a bal
ance between eRP and RP, Lumimaid was designed to be serious, yet uncensored when necessary.\n\nTo enhance it's overall intelligence and chat cap
ability, roughly 40% of the training data was not roleplay. This provides a breadth of knowledge to access, while still keeping roleplay as the p
rimary strength.\n\nUsage of this model is subject to [Meta's Acceptable Use Policy](https://llama.meta.com/llama3/use-policy/).", 'context_lengt
h': 8192, 'architecture': {'modality': 'text->text', 'input_modalities': ['text'], 'output_modalities': ['text'], 'tokenizer': 'Llama3', 'instruc
t_type': 'llama3'}, 'pricing': {'prompt': '0.000004', 'completion': '0.000006', 'request': '0', 'image': '0', 'audio': '0', 'web_search': '0', 'i
nternal_reasoning': '0'}, 'top_provider': {'context_length': 8192, 'max_completion_tokens': 4096, 'is_moderated': False}, 'per_request_limits': N
one, 'supported_parameters': ['frequency_penalty', 'max_tokens', 'min_p', 'presence_penalty', 'repetition_penalty', 'seed', 'stop', 'temperature'
, 'top_k', 'top_p']}

**Comment by justplus** [comment]
能提供完整的日志吗？或者复现步骤，我这边暂时没遇到你说的“配置了本地vllm部署的模型，但是实际评估的时候没有使用”的情况，我们评估也是本地vllm的模型。方便的话提供下复现步骤。

**Comment by 2niuhe** [comment]
还在摸索试用，环境拆掉了，问题先关闭了，感谢。
